js/plugins/vertexai/src/gemini.ts

```diff
@@ -507,7 +507,8 @@ export function defineGeminiModel(
       }
       // Cannot use tools and function calling at the same time
       const jsonMode =
-        (request.output?.format === 'json' || !!request.output?.schema) &&
+        (request.output?.format === 'json' || !!request.output?.schema ||
```

Comment:
I'm not sure the issue should be solved at the plugin level. 
I'm thinking perhaps somewhere here: https://github.com/firebase/genkit/blob/4e53051b0e18aad185d7b5709262242bf50f0a3f/js/ai/src/generate.ts#L249 
I'm looking into solving this at genkit `generate` level.

Reply:
that works too, I created a PR in here too: https://github.com/TheFireCo/genkit-plugins/pull/161